src/components/NavBar/_NavBar.scss

```diff
@@ -0,0 +1,173 @@
+.nav-bar{
+	width: 100%;
+	height: 7vh;
+	background-color: #143A52;
+	display: flex;
+	cursor: pointer;
+	&__menu{
+		width: 15%;
+		height: 6vh;
+		cursor: pointer;
+		@include large{
+			display: none;
+		}
+		&__button{
+			width: 100%;
+			height: 100%;
+			border: none;
+			background: no-repeat;
+			& > :first-child{
+				border: 1px solid;
+				width: 25px;
+				margin-bottom: 5px;
+				margin-left: 20%;
+				color: white;
+				transition: transform 2.0s;
+				&.first-element--active{
+					transform: rotate(-30deg) scale(.5) translateX(-14px);
+				}
+			}
+			& > :nth-child(2){
+				border: 1px solid;
+				width: 25px;
+				margin-bottom: 5px;
+				margin-left: 20%;
+				color: white;
+			}
+			& > :last-child{
+				border: 1px solid;
+				width: 25px;
+				margin-bottom: 5px;
+				margin-left: 20%;
+				color: white;
+				transition: transform 2.0s;
+				&.last-element--active{
+					transform: rotate(30deg) scale(.5) translateX(-14px);
+				}
+			}
+		}
+	}
+	&__image-container{
+		width: 30%;
+		& > img{
+			width: 100%;
+			height: 100%;
+			object-fit: scale-down;
+		}
+	}
+	&__list{
+    display: flex;
+    position: absolute;
+    flex-direction: column;
+		width: 100%;
```

Comment:
Indentation seems to be wrong in a few files.

Reply:
![image](https://user-images.githubusercontent.com/33492742/35833463-5e5f5a7c-0a9f-11e8-9a90-e7e52319ebd4.png)
Here as you can see in my file editor i don't notice any indentation problems, what should i do?